api/schema/types/ContributorType.js

```diff
@@ -5,6 +5,7 @@ module.exports = gql`
     type Contributor {
         id: Int!
         toggl_id: Int
+        total_paid: Int!
```

Comment:
One pattern here I would recommend that I like to do is separate properties that correspond to a column in the DB from the custom computed properties like this one or even from relationships. This way it is not confusing just by looking at the Contributor type I might think that `total_paid` is an actual database column that would need to be constantly updated.

Example:

```
type Contributor {
    // Columns
    
    id: Int!
    toggl_id: Int
    name: String!
    external_data_url: String!
    ...
    
    // Computed properties/relations
    
    total_paid: Int!
    allocations: [Allocation]
    permissions: [Permission]
    timeEntries: [TimeEntry]
    rates: [Rate]
}

```